### PR TITLE
Missing comma and array brackets

### PR DESCRIPTION
**Pull request by elmorec**

### CODE REVIEW

File: src/content/configuration/index.md [hunk 1]
```diff
@@ -183,8 +183,7 @@ module.exports = {
       { [resource](/configuration/module#rule-resource): { [or](/configuration/module#condition): [ /* conditions */ ] } },
       { [resource](/configuration/module#rule-resource): [ /* conditions */ ] },
       // matches if any condition is matched (default for arrays)
-      { [resource](/configuration/module#rule-resource): { [not](/configuration/module#condition): /* condition */ } }
-      // matches if the condition is not matched
+      { [resource](/configuration/module#rule-resource): { [not](/configuration/module#condition): [ /* condition */ ] } }
```
Comment (montogeek) [inline src/content/configuration/index.md]: Why did you delete the comment?

Comment (elmorec) [reply src/content/configuration/index.md]: sry, mistake...
